incubator/hnc/docs/releasing.md

```diff
@@ -112,9 +112,18 @@ this](https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-se
 ## Test
 
 Test! At a minimum, install it onto a cluster, download the kubectl plugin, and
-ensure that you can create a subnamespace. Better yet, run through the demo and
-all the `hack/test-*.sh` scripts. _TODO: automate this
-[#785](https://github.com/kubernetes-sigs/multi-tenancy/issues/785)_
+run:
+
+```bash
+export HNC_REPAIR=<path to installable YAML; https is fine>
+make e2e-test
+```
+
+This will take around 15m or so. Note the `-timeout 0` flag, which disables the
+default Go test timeout of 10m.
```

Comment:
Sorry that this was for the previous `-timeout 0` that I had question too. What is the default 10m timeout? Is it for the entire `go test`?

Reply:
Yes, it's for the entire test suite. Added documentation to the makefile and removed it from releasing.md, where it no longer makes sense.